bin/docker-worker-celery

```diff
@@ -64,6 +64,7 @@ if [ "$with_scheduler" == "true" ]; then
 fi
 
 FLAGS=()
+FLAGS+=("-Ofair")
```

Comment:
Can we please also add this to `bin/start-worker` so that discrepancies / unexpected surprised between local dev and prod are minimized?

Reply:
Good catch, done!